frontend/apps/reader/modules/readerstyletweak.lua

```diff
@@ -446,12 +447,16 @@ function ReaderStyleTweak:onSaveSettings()
 end
 
 local function dispatcherRegisterStyleTweak(tweak_id, tweak_title)
-    Dispatcher:registerAction(ReaderStyleTweak.dispatcher_prefix..tweak_id,
-        {category="none", event="ToggleStyleTweak", arg=tweak_id, title=T(_("Toggle style tweak: %1"), tweak_title), rolling=true})
+    Dispatcher:registerAction(ReaderStyleTweak.dispatcher_prefix_set..tweak_id,
+        {category="string", event="ToggleStyleTweak", arg=tweak_id, title=T(_("Style tweak '%1' set"), tweak_title), rolling=true,
```

Comment:
You think it's unclear without the "set"?

Reply:
For consistency with "toggle". Maybe, not needed.